src/libmongoc/src/mongoc/mongoc-timeout.c

```diff
@@ -20,7 +20,7 @@
 typedef struct _mongoc_timeout_t {
    bool is_set;
    int64_t timeout_ms;
-} mongoc_timeout_t;
+};
```

Comment:
This change produces a build failure for me on MacOS:
```
[ 43%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-topology.c.o
/Users/alcaeus/Code/mongodb/mongo-c-driver/src/libmongoc/src/mongoc/mongoc-timeout.c:20:1: error: typedef requires a name [-Werror,-Wmissing-declarations]
typedef struct _mongoc_timeout_t {
^~~~~~~
1 error generated.
make[2]: *** [src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-timeout.c.o] Error 1
```

Reply:
Submitted a PR for this here: https://github.com/mongodb/mongo-c-driver/pull/746. Can you confirm it works locally?

Reply:
Approved the PR after confirming it works. Thanks for the quick turnaround!